CHANGELOG.md

```diff
@@ -2,7 +2,15 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Added
+
+### Fixed
+- Replaced deprecated 'form' type by FormType class
+
+
```

Comment:
We changed them recently, so it's possible you have outdated instructions in mind : we realized asking everybody to edit just one file at the same § would lead to endless conflicts.